### ISSUE TITLE
Update Checkout to a more professional look

### ISSUE THREAD

**Issue by piajesse** [AccelerateNetworks/NumberSearch#317]
**How will this improve revenue or reduce our costs? Please provide a complete explanation!**
I've received word that our checkout portion of our website is unprofessional, so I've redone the form fill in text and explanations. 

**Describe the solution you'd like**
so far I have produced the following lines in [Order.cshtml](https://github.com/AccelerateNetworks/NumberSearch/blob/632bc8f93b5c5541e30a45c560756a2de99a4cf6/NumberSearch.Mvc/Views/Cart/Order.cshtml#L545)
```html

<form method="post" action="/Cart/Submit">
                
    <input type="text" value="26f0dacc-3825-4ddf-b58b-cc4ea79863d1" data-val="true" data-val-required="The OrderId field is required." id="Order_OrderId" name="Order.OrderId" hidden="">
    <div class="row">
        <div class="col-md-6 mb-3">
            <label for="firstName">First name</label>
            <input type="text" class="form-control" id="firstName" required="" data-val="true" data-val-required="The FirstName field is required." name="Order.FirstName" value="" placeholder="First">
            
        </div>
        <div class="col-md-6 mb-3">
            <label for="lastName">Last name</label>
            <input type="text" class="form-control" id="lastName" required="" data-val="true" data-val-required="The LastName field is required." name="Order.LastName" value="" placeholder="Last">
            
        </div>
    </div>

    <div class="mb-3">
        <label for="email">Business Name <span class="text-muted">(Optional)</span></label>
        <input type="text" class="form-control" id="businessName" data-val="true" data-val-required="The BusinessName field is required." name="Order.BusinessName" value="" placeholder="Company">
        <small class="text-muted">
            Name of your business or the organization that you represent.
        </small>
    </div>

    <div class="mb-3">
        <label for="email">Email </label>
        <input type="email" class="form-control" id="email" required="" data-val="true" data-val-required="The email field is required." name="Order.Email" value="" placeholder="Email">
        <small class="text-muted">
            For shipping updates and order information.
        </small>
    </div>

        <div class="mb-3" id="searchForm">
            <label for="address">Address</label>
            <input type="text" class="form-control" id="address" pattern=".+[,].+" oninput="handleSearchQuery(this)" onfocus="handleSearchQuery(this)" list="addressOptions" autocomplete="off" data-toggle="dropdown" required="" data-val="true" data-val-required="The UnparsedAddress field is required." name="Order.UnparsedAddress" value="" placeholder="400 Broad Street, Seattle, WA 98109">
            <ul class="list-group" role="menu" aria-labelledby="menu1" id="addressOptions">
            </ul>
            <small class="text-muted">
                Shipping address for this order and/or the address of the installation location.</small>
        </div>

    <div class="row">
        <div class="col-md-4 mb-3">
            <label for="unitType">Unit Type <span class="text-muted">(Optional)</span></label>
            <select class="custom-select d-block w-100" id="unitType" data-val="true" data-val-required="The AddressUnitType field is required." name="Order.AddressUnitType">
                        <option>Apartment</option>
                        <option>Basement</option>
                        <option>Building</option>
                        <option>Department</option>
                        <option>Floor</option>
                        <option>Front</option>
                        <option>Hanger</option>
                        <option>Lobby</option>
                        <option>Lot</option>
                        <option>Lower</option>
                        <option>Office</option>
                        <option>Penthouse</option>
                        <option>Pier</option>
                        <option>Rear</option>
                        <option>Room</option>
                        <option>Side</option>
                        <option>Space</option>
                        <option>Suite</option>
                        <option>Trailer</option>
                        <option>Unit</option>
                        <option>Upper</option>
            </select>
            
        </div>
        <div class="col-md-8 mb-3">
            <label for="address2">Unit Number <span class="text-muted">(Optional)</span></label>
            <input type="text" class="form-control" id="address2" data-val="true" data-val-required="The AddressUnitNumber field is required." name="Order.AddressUnitNumber" value="" placeholder="">
            
        </div>
    </div>
    <br>
    <h4 class="mb-3">Installation Information</h4>
    <div class="row">
        <div class="col-md-6 mb-3" id="installDateContainer">
            <label for="installdate">Preferred Install Date and Time <span class="text-muted">(Optional)</span></label>
                    <input type="datetime-local" class="form-control" id="installdate" readonly="" name="Order.InstallDate" value=""><input name="__Invariant" type="hidden" value="Order.InstallDate">
        </div>
        <div class="col-md-6 mb-3">
            <label for="sales">How did you hear about us? <span class="text-muted">(Optional)</span></label>
            <select class="custom-select mr-sm-2" id="sales" data-val="true" name="Order.SalesEmail" data-val-required="The Sales Email field is required.">
                <option value="" selected="">Choose...</option>
                <option value="scott@acceleratenetworks.com">Scott</option>
                <option value="tyler@acceleratenetworks.com">Tyler</option>
                <option value="dan@acceleratenetworks.com">Dan</option>
                <option value="jesse@acceleratenetworks.com">Jesse</option>
                <option value="thad@acceleratenetworks.com">Thad</option>
                <option value="youtube@acceleratenetworks.com">Youtube</option>
                <option value="blog@acceleratenetworks.com">Our Blog</option>
                <option value="mail@acceleratenetworks.com">Mail</option>
                <option value="referred@acceleratenetworks.com">Referred... (Please explain in Notes)</option>
                <option value="other@acceleratenetworks.com">Other... (Please explain in Notes)</option>
            </select>
            
        </div>
    </div>
    <div class="row">
        <div class="col-md-12 mb-3">
            <label for="customernotes">Notes <span class="text-muted">(Optional)</span></label>
            <input type="text" class="form-control" id="customernotes" placeholder="Add important information for our staff" data-val="true" data-val-required="The CustomerNotes field is required." name="Order.CustomerNotes" value="">
        </div>
        <div class="col-md-12">
            <div class="form-check">
                <input type="checkbox" class="form-check-input" id="defaultCheck1" data-val="true" data-val-required="The Quote field is required." name="Order.Quote" value="true">
                <label class="form-check-label" for="defaultCheck1">
                    Please send a quote
                </label>
            </div>
        </div>
    </div>
            <hr class="mb-4">
            <button class="btn btn-primary btn-lg btn-block shadow" type="submit" value="Submit">Submit your Order</button>
    <br>
<input name="__RequestVerificationToken" type="hidden" value="CfDJ8CQ2v3PW0zxIj6gKB1ezIUYebrEK8PVDI1hiS1iol_cukd39wmHjuYwZAxCXeaTaoa_3JRDz4g9xJu6hboEsF53HqlLt1JltubHxJCpl1dHFt23lyhJBTbmMFaFW1fZy5T9lO4M9iIchn8LLJXwv_hQ"><input name="Order.Quote" type="hidden" value="false"></form>
```
**Additional context**
It would also be nice to separate the address input form to individual forms for State, Zip, etc. If that can't be done, I left the address of the space needle as an example in placeholder format. I was gonna do this and the changes above myself, but Dan assigned me back to working on the apps.

**Comment by uncheckederror** [comment]
Yeah, it's difficult for me to tell what you've changed in the above code block. Can you submit this as a PR to the https://github.com/AccelerateNetworks/NumberSearch/blob/master/NumberSearch.Mvc/Views/Cart/Order.cshtml file? Thanks.

As a separate issue, if someone finds our checkout page "unprofessional" then they are not the kind of customer that we want to attract, as they have already demonstrated that they are needy and high maintenance. Was this person having a bad day? Hugs are free.

On the additional context: I'm not saying no, but why would separating the address inputs with additional State and Zip fields be superior to the current single field? We actually had the address inputs as separate fields up until about 2 years ago, but the live address validation is less frustrating to use if it only updates a single field rather than multiple fields. When the form is submitted we maintain both formats in the database where we have the address as a single line as shown on the form and broken out into House Number, Street, State and Zip fields. 

The Unit Type and Unit Number fields are separate because they are not supported by the address validation API we're using and they need to enumerate a specific set of options so we can register the customer's address for E911 service.

**Comment by danry25** [comment]
>As a separate issue, if someone finds our checkout page "unprofessional" then they are not the kind of customer that we want to attract, as they have already demonstrated that they are needy and high maintenance. Was this person having a bad day? Hugs are free.

The checkout page looks decent, but it needs some touch ups.
- Description Text "Enter your first name (ex. Alexander)"
  - Should become "Enter your first name" since we already fill the field in with example text
- Enter your last name (ex. Bell) 
  - Should become "Enter your last name" since we already fill the field in with example text
- Business Name example text is misspelled as American Telegraph and Telephone
  - A more generic name like ACME Corporation would be ideal (and a better in joke :wink: )
- Unit Type description text should drop "(ex. Apartment)"
- Unit Number description text should drop (ex. 307)

**Comment by uncheckederror** [comment]
These changes are live.

**Comment by uncheckederror** [comment]
Updated the email list at @Chattaeus's request. 

The purpose of this Sales rep field to organize information in the Ops app, so that the different Sales reps only have to sort through the Quotes and Orders that they are responsible for rather than sorting through everything like the support staff have to. Hence in the list is limited to just the Sales reps by name. I have added Support as a catch-all for everything that is non-sales, although I'm not entirely sure that it makes sense as we can't use it to organize anything inside the Ops app.
